### DISCUSSION TITLE
Implement support for RCCB bayer pattern demosaicing.

### DISCUSSION THREAD

**Discussion by raviy0807** [colour-science/colour-demosaicing#32] [Ideas]
There is good amount of implementation for different bayer pattern. But I didn't find any api/function for demosaicing RCCb or RCCC. Could someone suggest on it? Can we raise feature request for this?

**Comment by thomasmansencal** [comment]
Hi @raviy0807,

No sorry those are not supported but it would be great to do so if people have the need for it. Do you have any references handy for it?

Cheers,

Thomas

**Reply by raviy0807** [reply]
Hi @KelSolaar 

I will try to find the reference. From a quick scan, I didn't find any papers or examples so far. But I will keep looking for it.
